frontend/test/unit/fixtures/audio.js

```diff
@@ -1,5 +1,11 @@
 import * as uuid from "uuid"
+/** @import { ApiMedia } from "~/types/media" */
```

Comment:
Would be nice to also have a fixture for `AudioDetail` (processed item with the extra properties added).

Reply:
I would! I'll make an issue or comment somewhere for having complete, type-safe fixtures for every media type.